src/extension/VSProject.cs

```diff
@@ -187,17 +187,23 @@ private void Load()
                 ThrowInvalidFileType( ProjectPath );
 
             StreamReader rdr = new StreamReader(ProjectPath, System.Text.Encoding.UTF8);
-            
+
             try
             {
                 _doc = new XmlDocument();
-                _doc.Load( rdr );
```

Comment:
Try to avoid making irrelevant changes to layout. It makes it slightly harder to review the PR because it gives us more lines of changes.